docs/BuildFromSource.md

```diff
@@ -322,7 +322,7 @@ Common properties include:
 
 ### Resx files
 
-If you need to make changes to a .resx file, run `dotnet msbuild t:/Resgen <path to csproj>`. This will update the generated C#.
+If you need to make changes to a .resx file, run `dotnet msbuild /t:Resgen <path to csproj>`. This will update the generated C#.
```

Comment:
@JunTaoLuo this is no longer true, isn't it?

Reply:
?

Reply:
ResGen is no longer a command we have to run. I think this section needs to be removed

Reply:
How does it work outside of VS?

Reply:
PIng @JunTaoLuo

Reply:
This target exists but it's not clear to me whether it works w/ anything but `@(EmbeddedResource)` items. I remember another (newer❔) target name but it's not coming to me. Will come back when I remember or find it&hellip;

Reply:
@dougbu Any updates here?

Reply:
This entire section does not apply to `*.resx` files in the 'release/3.1', 'release/5.0', 'release/6.0-preview7', or 'main' branches. The source files are not checked into these branches and, as far as I can tell, aren't written to disk. (The [src/Mvc/Mvc.Localization/src/Properties/Resources.Designer.cs](https://github.com/dotnet/aspnetcore/commits/main/src/Mvc/Mvc.Localization/src/Properties/Resources.Designer.cs) file has no corresponding Resources.resx.)

Generated Resources.designer.cs files still exist in 'release/2.1'. But, the target name to generate them is `Resx` and not `Resgen`.

I suspect we used `Resgen` (which is part of the .NET SDK) at some point to update Resources.designer.cs files on disk at some point. Since then, we threw the whole thing away and rely on `Resgen` to do it's default thing and automatically include the requisite classes in our assemblies. Comments in a few of the 'release/2.1' generated files indicate the `Resgen` tool has a `/str` option but I don't see a way to use that from `msbuild`.

Reply:
```suggestion
After making changes to a .resx file, the updated strings and accessor methods will automatically be included in the output assembly when the project is next built.
```